packages/create-hydrogen/package.json

```diff
@@ -13,7 +13,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@shopify/cli-hydrogen": "^7.1.0"
+    "@shopify/cli-hydrogen": "latest"
```

Comment:
(Moving here for threading)

> But unsure if we can make this work without installing a remote version in our monorepo with `latest` 🤔

@frandiox Hm, yeah. I'm not sure what's causing the failure in CI here. But the `latest` tag is [published by default](https://docs.npmjs.com/adding-dist-tags-to-packages) and it's always [available to install](https://www.npmjs.com/package/@shopify/cli-hydrogen?activeTab=versions) through npm:

![image](https://github.com/Shopify/hydrogen/assets/547470/5d7965c8-1009-4d4e-b358-233965537583)

Reply:
Ah right. Done!

Reply:
😂 😭 
Alsooo... did it break a bunch of stuff? I see two failed actions, they seem to be related to the node version? 🤔 
- https://github.com/Shopify/hydrogen/actions/runs/8378036152
- https://github.com/Shopify/hydrogen/actions/runs/8378036152

Reply:
I wonder if something like [link](https://github.com/changesets/changesets/blob/main/docs/linked-packages.md) from chageset would help here